app/src/controllers/fetch/reducer.js

```
@@ -23,35 +23,37 @@
+        } else {
+          const selectedPayload =
+            contentPath && payload[contentPath] ? payload[contentPath] : payload;
+          return payloadConverter ? payloadConverter(selectedPayload) : selectedPayload;
+        }
+      }
+      case CONCAT_FETCH_SUCCESS: {
+        const data = contentPath ? payload[contentPath] : payload;
 
-      if (data instanceof Array && concat) {
-        return state.concat(data);
+        if (data instanceof Array && concat) {
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)